src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -30,7 +30,7 @@ internal partial class Binder
         // If we have no modifiers then the modifiers array is null; if we have any modifiers
         // then the modifiers array is non-null and not empty.
 
-        private Tuple<ImmutableArray<RefKind>, ImmutableArray<TypeSymbol>, ImmutableArray<string>, bool> AnalyzeAnonymousFunction(
+        private Tuple<ImmutableArray<RefKind>, ImmutableArray<TypeSymbol>, ImmutableArray<string>, ImmutableArray<bool>, bool> AnalyzeAnonymousFunction(
```

Comment:
I'm just going to switch these to out parameters.  Not much sense having a 1-off tuple just to package up the results and then immediately unpack them.